app/routes/account.$accountId.settings.members/action.ts

```diff
@@ -30,6 +37,7 @@ export const action: ActionFunction = async ({ request, params }) => {
     const user_resend = formData.get("user_resend")
     const account_name = formData.get("account_name") as string
     const invitedAccountName = account_name ? account_name : accountId
+    const ipAddress = getClientIPAddress(request)
```

Comment:
we should be able to do this with just the user_id and not need the IP

Reply:
or account_id

Reply:
That how I did it initially, I though of using the ip because it was recommended but indeed I think the `account_id`/`user_id` is enough.

Reply:
Can you think of other requests where we should be rate limiting as well? @kutoft